region/lowernorfair/west/Fast Ripper Room.json

```diff
@@ -1082,6 +1255,73 @@
       "bypassesDoorShell": true,
       "flashSuitChecked": true
     },
+    {
+      "link": [2, 2],
+      "name": "R-Mode Spark Interrupt (Gain Blue Suit)",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        "h_heatedGreenGateGlitch",
+        {"or": [
+          {"and": [
+            {"or": [
+              {"resourceAvailable": [{"type": "Super", "count": 10}]},
+              {"and": [
+                {"or": [
+                  "ScrewAttack",
+                  {"enemyKill": {"enemies": [["Ripper 2 (red)", "Ripper 2 (red)"]], "explicitWeapons": ["PowerBomb"]}}
+                ]},
+                {"heatFrames": 600},
+                {"partialRefill": {"type": "Super", "limit": 10}}
```

Comment:
You can't refill to 10 if you take 10 tries on the gate glitch.
It must be hard to refill 6.
And the powerbomb says you kill 2, so this sequence is a bit misleading and looks like it would want `First-Try Gate Glitch`